templates/includes/records/record-cta-panel.html

```diff
@@ -0,0 +1,17 @@
+<div class="cta-primary-panel">
+    <div class="container" data-id="cta-toggle-link">
+        <a class="cta-primary-panel__link cta-primary-panel__link--secondary" href="/search">Back to search results</a>
+        {% if not FEATURE_RECORD_LINKS_GO_TO_DISCOVERY %}
+        {% endif %}
+    </div>
+</div>
+
+{# {% if FEATURE_RECORD_LINKS_GO_TO_DISCOVERY %} #} <!-- delete the jinja comment blocks to enable this-->  
+<div class="cta-secondary-panel" data-id="cta-detailed-view">
+    <div class="container">
+        <h2 class="cta-secondary-panel__heading">Looking to view, download or order a copy of this record?</h2>
+        <p class="cta-secondary-panel__text">Access options are not available in this service. To find out whether you can view, order or download this record you will need to view the availability of this record on it's description page in Discovery - our catalogue.</p>
+        <a class="cta-secondary-panel__link" target="_blank" href="https://discovery.nationalarchives.gov.uk/details/r/{{page.template.iaid}}">View this record in Discovery <span class="sr-only">(link opens in a new window)</span></a>
```

Comment:
#FEATURE_RECORD_LINKS_GO_TO_DISCOVERY
If the implementation is in question. 
Will need to refactor to say RECORD_LINKS_GO_TO_DISCOVERY.
https://github.com/nationalarchives/ds-wagtail/blob/develop/templates/insights/blocks/featured_record.html#L34
`href="{% record_url value.record is_editorial=True %}"` 
something like 
href="{% record_url page is_editorial=True %}"